samples/constants.go

```diff
@@ -5,8 +5,18 @@ import "fmt"
 
 const Phi = 2.61803398875
```

Comment:
This is phi + 1?

Reply:
`1.618033988749895`

see: https://goplay.space/#fwTUCXZBO_6

Reply:
oups, thanks.

Reply:
Done.